addons/tmf_loadouts/loadouts/us_rangers_2010.hpp

```diff
@@ -0,0 +1,610 @@
+		LIST_2("MRAWS_HEAT_F"),
+		"MRAWS_HE_F"
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"B_Kitbag_sgg"};
+	backpackItems[] = {
+		LIST_3("MRAWS_HEAT_F"),
+		LIST_2("MRAWS_HE_F")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"B_Kitbag_sgg"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_2("MRAWS_HEAT_F"),
+		"MRAWS_HE_F"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Tow_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_TOW2_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_TOW2_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR ,"ACE_RangeTable_82mm","ACE_MapTools"};
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"Rangefinder",
+		"ItemGPS"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	uniform[] = {"CFP_75th_CRYE_V4_Full"};
+	vest[] = {"CUP_V_I_RACS_Carrier_Vest_wdl"};
+	backpack[] = {"B_Kitbag_rgr"};
+	headgear[] = {"H_HelmetB_camo"};
+	hmd[] = {"CUP_NVG_PVS15_black"};
+	primaryWeapon[] = {"CUP_arifle_M4A1_standard_short_black"};
+	scope[] = {"cup_optic_eotech553_coyote"};
+	silencer[] = {"cup_muzzle_snds_m16_coyote"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("HandGrenade"),
+		"ACE_HandFlare_Red",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+	backpackItems[] = {
+		"CUP_optic_LeupoldMk4_10x40_LRT_Desert",
+		"CUP_bipod_Harris_1A2_L_BLK",
+		LIST_8("CUP_5Rnd_762x51_M24"),
+		"CFP_75th_Beanie_Grn",
+		"CUP_srifle_M24_des"
```

Comment:
If you check the config viewer, there are rifles with attachments already attached (hidden in arsenal). Alternatively, you can check the CUP wiki: http://wiki.audiocustoms.info/index.php?title=Arma_3_CfgWeapons_Weapons

Reply:
Thank you for the fantastic resource! I've replaced the mess with "CUP_srifle_M24_des_LeupoldMk4LRT" instead. Should clean up the backpack a good amount.